Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/LightingUtils.azsli

```diff
@@ -50,10 +50,10 @@ float GetRoughnessMip(float roughness)
     return roughness * maxRoughnessMip;
 }
 
-// compute parallax corrected reflection vector
+// compute parallax corrected reflection vector, AABB version
 // we do this by finding the intersection with the volume and adjusting the reflection vector for the surface position 
 // https://seblagarde.wordpress.com/2012/09/29/image-based-lighting-approaches-and-parallax-corrected-cubemap/
-float3 ApplyParallaxCorrection(float3 aabbMin, float3 aabbMax, float3 aabbPos, float3 positionWS, float3 reflectDir)
+float3 ApplyParallaxCorrectionAABB(float3 aabbMin, float3 aabbMax, float3 aabbPos, float3 positionWS, float3 reflectDir)
```

Comment:
Shouldn't we remove `positionWS` and just stipulate that `aabbMin` and `aabbMax` have the world space offset applied?

Reply:
The positionWS input is the surface position for the parallax correction, so we can't eliminate it.  If you mean the aabbPos, we need it to compute the offset, so I'd prefer to keep it as an input rather than spending ALU to compute it from the min/max.